pkg/operator/controller/controller.go

```diff
@@ -92,16 +92,20 @@ type reconciler struct {
 	Config
 }
 
-// Reconcile currently performs a full reconciliation in response to any
-// request. Status is recomputed and updated after every request is processed.
-//
-// TODO: Refactor the way the controllers are set up; the only requests that
-// should be coming into this controller are clusteringress keys. Then the loop
-// can act on a single clusteringress. To achieve this, watch events from
-// related resources must be translated into clusteringress keys through label
-// checks and such (as various relationships cross boundaries which can't be
-// expressed by ownerrefs).
+// Reconcile expects request to refer to a clusteringress in the operator
+// namespace, and will do all the work to ensure the clusteringress is in the
+// desired state.
 func (r *reconciler) Reconcile(request reconcile.Request) (reconcile.Result, error) {
+	result, err := r.reconcile(request)
+	if err != nil {
+		// TODO: Figure out why the controller-runtime logger impl isn't logging
+		// these.
+		logrus.Errorf("error: %v", err)
```

Comment:
I figured this out, but I think it'll need fixed in a followup. We aren't wiring in a controller-runtime logr implementation, so all controller-runtime logs are getting sent to devnull...